packages/core/src/configuration.ts

```diff
@@ -153,8 +153,9 @@ const chart: BaseChartOptions = {
 		loading: false,
 	},
 	color: {
-		scale: null,
+		scale: null
 	},
+	selectedGroups: []
```

Comment:
should go into the `data` field within options

Reply:
Ok, I'll revise that.